.github/workflows/setupWlsAks.yml

```diff
@@ -36,7 +36,7 @@ env:
 jobs:
   # Make it so the bicep file that causes WLS on AKS to be deployed is available to this workflow.
   preflight:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Very good. I dislike latest, for this reason. https://stackoverflow.com/questions/68991701/should-we-specify-the-version-of-the-base-docker-image-in-dockerfiles